pages/common/printf.md

```diff
@@ -22,3 +22,7 @@
 - Store a formatted message in a variable (does not work on zsh):
 
 `printf -v {{myvar}} {{"This is %s = %d\n" "a year" 2016}}`
+
+- Try printing other format types:
```

Comment:
I think we would use only the ones, that are missing. e.g  `octal`, `hex`, `scientific`, because rest are used in other examples.
```suggestion
- Print a hexadecimal, octal and scientific number:
```

What do you think ?

Reply:
I am fine with that. I will push a commit to change the actual example.